exercises/dominoes/dominoes.rb

```diff
@@ -0,0 +1,53 @@
+class Dominoes
+  require_relative 'domino'
+  attr_accessor :list
+
+  #[ArrayTuples], false => [Domino] or [Domino], true => [Domino]
+  def initialize( arr=[], already_dominoes= nil)
+    @list = []
+    arr.each do |item|
```

Comment:
Rubyism: if you use `map` instead of `each`, you don't need the `@list` variable.